nbactions.xml

```diff
@@ -24,7 +24,7 @@
                 <goal>org.codehaus.mojo:exec-maven-plugin:1.5.0:exec</goal>
             </goals>
             <properties>
-                <exec.args>-agentlib:jdwp=transport=dt_socket,server=n,address=${jpda.address} -classpath %classpath ${packageClassName}</exec.args>
+                <exec.args>-agentlib:jdwp=transport=dt_socket,server=n,address=${jpda.address} -classpath %classpath apps.PanelPro.PanelPro</exec.args>
```

Comment:
I think this change needs to be reverted.

Reply:
sorry about  that. Don't know how it happened. Don't have any record of changing it. Have changed it back.

Reply:
Thanks! I think it's a NetBeans file that NetBeans creates.